cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/AbstractClouderaManagerApiCheckerTask.java

```diff
@@ -101,6 +103,17 @@ private boolean isToleratedError(ApiException e) {
                 || e.getCause() instanceof SocketTimeoutException;
     }
 
+    protected String getResultMessageWithDetailedMessagePostFix(ApiCommand apiCommand, CommandsResourceApi commandsResourceApi) {
```

Comment:
not sure, that's why I'm asking but what ensures that the apiCommand is always non-null? since one line below we're accessing it and that could cause an NPE

Reply:
What line do you mean?

But you are right, the NPE check worthy, in that case returns with an empty string.